app/views/orders/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2>Orders</h2>
+  <br>
+    <div class="row hstack gap-3">
+      <% @orders.each do |order| %>
+        <div class="card" style="width: 18rem">
+          <div class="card-body">
+            <h5 class="card-title"><%= order.reward_snapshot.title %></h5>
+            <p class="card-text"><%= order.reward_snapshot.description %></p>
+            <h6 class="card-subtitle mb-2 text-muted">Price: <%= order.reward_snapshot.price %></h6>
+            <h6 class="card-subtitle mb-2 text-muted">Purchase time: <%= order.created_at.strftime("%d-%m-%y %H:%M") %></h6>
```

Comment:
Maybe consider `using time_ag_in_words` helper for displaying time info https://apidock.com/rails/v6.0.0/ActionView/Helpers/DateHelper/time_ago_in_words

Reply:
I like to use I18n.l to format time. You could google `localize time format rails`